submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/experiment/codelocalizer/HTTPGitCodeLocalizer.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.submarine.server.submitter.k8s.experiment.codelocalizer;
+
+import io.kubernetes.client.models.V1PodSpec;
+
+public class HTTPGitCodeLocalizer extends GitCodeLocalizer {
```

Comment:
@manirajv06 , it looks like HTTP and SSH Git localizer pretty same, I'm wondering why we need the two implementations. And why `localize` of `SSHGitCodeLocalizer` is empty?

Reply:
As of today, it seem like there are no differences. But when we get into "git ssh mode" implementation, we will need this separation. For ex, GIT_SYNC_SSH needs to be set as "true".

Reply:
gotcha, make sense.